polymer/foam-storage.html

```diff
@@ -0,0 +1,59 @@
+<dom-module id="foam-storage">
+  <script>
+    (function() {
+      'use strict';
+      Polymer({
+        is: 'foam-storage',
+        hostAttributes: {
+          hidden: true
+        },
+
+        properties: {
+          model: {
+            type: Object
+          },
+          name: {
+            type: String,
+          },
+          type: {
+            type: String
+          },
+          value: {
+            type: Array,
+            notify: true
+          }
+        },
+
+        observers: [
+          'onValueUpdate(value.*)'
+        ],
+
+        created: function() {
+          apar(
+            arequire('foam.dao.EasyDAO'),
```

Comment:
arequire is now a method on the context.  This will work, but it would be more correct to do X.arequire